components/style/style_adjuster.rs

```diff
@@ -28,6 +28,12 @@ impl<'a, 'b: 'a> StyleAdjuster<'a, 'b> {
         StyleAdjuster { style }
     }
 
+    /// Whether we should skip display item-based display fixups.
```

Comment:
Consider also copying the comment from `SKIP_ROOT_AND_ITEM_BASED_DISPLAY_FIXUP` mentioning what would be affected by the returned value.

Reply:
Done.